packages/dx-react-chart/docs/guides/getting-started.md

```diff
@@ -0,0 +1,83 @@
+# React Chart - Getting Started
+
+## Overview
+Devextreme React Chart is a data visualization component with bar, line, stacked, area, scatter, pie and other series. 
+It has axis plugin for display scale and legend plugin for a description of series. 
+Now, Devextreme React Chart is the stateless component and depends on properties and working in a controlled mode only.
```

Comment:
'Devextreme' -> 'DevExtreme'

> component and depends on properties and working in a controlled mode only

Can't get much sense. Do you mean: "component **that** depends on properties and work**s** in the controlled mode only"?